content/components/cards.mdx

```diff
@@ -19,8 +19,8 @@ icon: "rectangle"
 ```
 
 ```jsx Image Card Example
-<Card title="Image Card" img="/images/card-with-image.png">
-  Here is an example of a card with an image
+<Card title="Image Card" img="/images/images/moo-deng.png">
+  Images can improve clarity (or bring delight) in your docs
```

Comment:
I love moo deng, but can we get a more gradient image from James so it's a bit more on brand? Alternatively, we've used hd images from national parks in other places

Reply:
sorry I did not mean to merge moo Deng I was working off an old incorrect test branch -- do not merge pls